webapp/src/index.js

```diff
@@ -66,11 +66,11 @@ export default class Plugin {
             return frontendListName;
         };
 
-        const refresh = ({data: {lists}}) => lists.forEach((listName) => store.dispatch(list(false, getFrontendListName(listName))));
+        const refresh = () => {
+            store.dispatch(fetchAllIssue());
```

Comment:
Can we have the websocket message that gets sent from the server contain the lists, rather than fetching them here?

Reply:
It's after

```
registry.registerWebSocketEventHandler(`custom_${pluginId}_refresh`, refresh);
```

Reply:
I'm not sure what this means. Are you saying it's not possible to deliver these changes in the websocket message?

Reply:
This line is the definition of the function that is then executed in the websocket.